src/components/navigation-header/navigation-header.module.css

```diff
@@ -99,7 +99,11 @@
 	}
 }
 
-.userDropdownDisclosure button[aria-expanded] {
+.userDropdownDisclosureActivator {
+	/* composition */
+	composes: g-focus-ring-from-box-shadow-dark from global;
```

Comment:
Is there a reason we wouldn't put these focus styles on the activator directly? Do we have a scenario where we need different focus styles?

Reply:
To add on to the general context I just left in reply to your other comment -- `DropdownDisclosure` is primarily used on light backgrounds outside of the dark navigation header. In this case, the `UserDropdownDisclosure` is sitting on a dark background, so it needs to have a different focus ring style.